src/Catalyst.Node.POA.CE/CatalystNodePoa.cs

```diff
@@ -0,0 +1,110 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Catalyst.Abstractions;
+using Catalyst.Abstractions.Consensus;
+using Catalyst.Abstractions.Contract;
+using Catalyst.Abstractions.Cryptography;
+using Catalyst.Abstractions.Dfs;
+using Catalyst.Abstractions.KeySigner;
+using Catalyst.Abstractions.Ledger;
+using Catalyst.Abstractions.Mempool;
+using Catalyst.Abstractions.P2P;
+using Catalyst.Abstractions.Types;
+using Catalyst.Core.Lib.DAO;
+using Serilog;
+using TheDotNetLeague.MultiFormats.MultiBase;
+
+namespace Catalyst.Node.POA.CE
+{
+    public class CatalystNodePoa : ICatalystNode
+    {
+        public IConsensus Consensus { get; }
+        private readonly IContract _contract;
+        private readonly IDfs _dfs;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_dfs' or refactor the code to use its value.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)